3.1_fixed.py

```diff
@@ -0,0 +1,57 @@
+class Book:
+    """ Базовый класс книги. """
+    def init(self, name: str, author: str):

```

Comment:
```suggestion
    def __init__(self, name: str, author: str):
```
Эти методы всегда включают нижние подчеркивания в названии!